docs/notes/notes.rst

```diff
@@ -47,11 +42,13 @@ If you are upgrading from a version older than 1.10.0, regenerating the code is
 Previous versions
 -----------------
```

Comment:
I'd put this title at the same level as the last version, i.e. `=====`. As it is now, on the toc tree, the "previous versions" section is placed inside the 2.0.0 version, it looks strange to me.